### PR TITLE
Leadership Change

### PR DESCRIPTION
**Pull request by chris-short**
Changes to policies where required due to change in co-organizers.

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -85,7 +85,6 @@ public domain, no credit and no open licensing of your version is required.
 * Please see the [Additional Resources][add-rec] document
 
 [organizers]: conduct-organizers.md
-[add-rec]: additional-resources.md
```
Comment (base10) [inline code-of-conduct.md:88]: I saw this was a dead link, but could you 1) retain the link and 2) update this to conduct-resources.md? Thanks

File: README.md [hunk 2]
```diff
@@ -5,15 +5,17 @@ Triangle DevOps organizers.
...
 ## Organizers
 
-* Mark Mzyk
-  * GitHub: mmzyk
-  * mmzyk (at) programmersparadox (dot) com
+* Chris Short
+  * GitHub: [chris-short](https://github.com/chris-short)
+  * Twitter: [@ChrisShort](https://twitter.com/ChrisShort)
+  * chris (at) chrisshort (dot) net
+* Chris Bailey
```
Comment (base10) [inline README.md]: If Chris Bailey is on the hot list to be contacted, can we get contact information here for him?

**Review by base10 — CHANGES_REQUESTED** [review]
I have two small change requests, please.

**Comment by chris-short** [comment]
All requests fulfilled

**Pull request merged**